pages/apis/rest_api/clusters.md

```diff
@@ -346,6 +346,138 @@ Required scope: `read_clusters`
 
 Success response: `200 OK`
 
+### Create a queue
+
+```bash
+curl -X POST "https://api.buildkite.com/v2/organizations/{org.slug}/clusters/{cluster.id}/queues" \
+  -H "Content-Type: application/json" \
+  -d '{ "key": "default", "description": "The default queue for this cluster" }'
```

Comment:
I thought a cluster already had a default queue when created? If so, perhaps this should create a queue with a different key and description to be more realistic?

Reply:
They do when created through the UI however we decided to change this behaviour for the API so it's more predictable and the user is in full control!